### DISCUSSION TITLE
Automatically include subtitle files for the media in the same directory

### DISCUSSION THREAD

**Discussion by atmos-o** [tks18/gindex-v4#239] [Ideas]
I always have subtitle files in the same directory level as the movie file, named the same. e.g. for the movie "Godzilla (2016)" the subtitle file would be `Godzilla (2016).en.srt` for an english sub file. I know othes have a separate `/Godzilla/subs/` folder with the subs in there too.

Would it be plausible to already have this included when going to view a movie instead of having to produce a link and adding it manually?

**Comment by tks18** [comment]
If the subtitle files are in the same directory as video file, that subtitle file will be included automatically whatever the format is srt or vtt, check again

**Reply by atmos-o** [reply]
I've checked with other Movies, I even renamed the subtitle file to match the video file completely, it still does not automatically include sub files.

**Reply by tks18** [reply]
See following titles will get automatically added to playback

1. [movie-name].[lang].srt/vtt
2. [movie-name].srt/vtt

Any number of Subtitle files will get added to playback
